src/rpc/handlers/GatewayBalances.cpp

```diff
@@ -180,6 +180,14 @@ doGatewayBalances(Context const& context)
             return obj;
         };
 
+    for (auto& hw : hotWallets)
```

Comment:
For better readability this could be replaced with something like:
```
auto isHotBalance = [&](auto const& hw) { return hotBalances.contains(hw); };
if (not all_of(begin(hotWallets), end(hotWallets), isHotBalance))
    return Status{Error::rpcINVALID_PARAMS, "invalidHotWallet"}; 
```
It's up to you which you prefer. Otherwise consider to use `auto const&` in the `for`.